mongoWellnessChecker/mongoWellnessChecker.js

```diff
@@ -0,0 +1,592 @@
+/* global db, tojson, tojsononeline, rs, print, printjson */
```

Comment:
```suggestion
/* global db, rs, print, printjson */
```

These aren't globals in mongosh by default

Reply:
ack